tests/common.go

```diff
@@ -5314,3 +5314,88 @@ func GetSubsetOfSlice[T any](items []T, length int) ([]T, error) {
 	}
 	return randomItems, nil
 }
+
+func DeleteGivenPoolInNode(stNode node.Node, poolIDToDelete string) (err error) {
+
+	log.InfoD("Setting pools in maintenance on node %s", stNode.Name)
+	if err = Inst().V.EnterPoolMaintenance(stNode); err != nil {
+		return err
+	}
+	//Waiting for cli to work
+	time.Sleep(2 * time.Minute)
+
+	if status, err := Inst().V.GetNodeStatus(stNode); err != nil {
+		return err
+	} else {
+		log.InfoD("Node [%s] has status: [%v] after entering pool maintenance", stNode.Name, status)
+	}
```

Comment:
Don't need else here, as it would return if there is an error

Reply:
fixed